From 328f8dd44fdadbd17d910705f105f7ecae6969bd Mon Sep 17 00:00:00 2001
From: Karthik M <quic_karm@quicinc.com>
Date: Mon, 17 Apr 2023 17:22:39 +0530
Subject: [PATCH] ath11k: Fix the wrong regdomain update for KR country

KR country should follow JP regulatory domain instead it
was assigned ETSI domain wrongly.

This issue got exposed after the change
5dbddc3d4ac ("cfg80211: precac check for self-managed wiphy")
got merged.

Signed-off-by: Karthik M <quic_karm@quicinc.com>
---
 drivers/net/wireless/ath/ath11k/reg.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/ath/ath11k/reg.c b/drivers/net/wireless/ath/ath11k/reg.c
index 2926553..0a3aaef 100644
--- a/drivers/net/wireless/ath/ath11k/reg.c
+++ b/drivers/net/wireless/ath/ath11k/reg.c
@@ -231,10 +231,10 @@ ath11k_map_fw_dfs_region(enum ath11k_dfs_region dfs_region)
 	case ATH11K_DFS_REG_CN:
 		return NL80211_DFS_FCC;
 	case ATH11K_DFS_REG_ETSI:
-	case ATH11K_DFS_REG_KR:
 		return NL80211_DFS_ETSI;
 	case ATH11K_DFS_REG_MKK:
 	case ATH11K_DFS_REG_MKK_N:
+	case ATH11K_DFS_REG_KR:
 		return NL80211_DFS_JP;
 	default:
 		return NL80211_DFS_UNSET;
-- 
2.17.1

